### ISSUE TITLE
Compiling Issue for windows

### ISSUE THREAD

**Issue by gr1mslasher** [ShadowMario/FNF-PsychEngine#11123]
### Describe your problem here.

Hello, I am having trouble compiling Psyche Engine. When I compile it, I get this message:
source/Achievements.hx:132: characters 30-54 : Warning : `FlxCamera.defaultCameras` is deprecated, use `FlxG.cameras.setDefaultDrawTarget` instead
source/MainMenuState.hx:69: characters 3-39 : Warning : `FlxCamera.defaultCameras` is deprecated, use `FlxG.cameras.setDefaultDrawTarget` instead
source/PlayState.hx:370: characters 3-39 : Warning : `FlxCamera.defaultCameras` is deprecated, use `FlxG.cameras.setDefaultDrawTarget` instead
source/editors/CharacterEditorState.hx:94: characters 3-41 : Warning : `FlxCamera.defaultCameras` is deprecated, use `FlxG.cameras.setDefaultDrawTarget` instead
source/editors/DialogueCharacterEditorState.hx:87: characters 3-40 : Warning : `FlxCamera.defaultCameras` is deprecated, use `FlxG.cameras.setDefaultDrawTarget` instead
source/options/NoteOffsetState.hx:57: characters 3-39 : Warning : `FlxCamera.defaultCameras` is deprecated, use `FlxG.cameras.setDefaultDrawTarget` instead
Sys_error("export/debug/windows/obj/Options.txt: Permission denied")

I know where the errors are located and in what line of text (hx:#) but I don't know how to fix it. It only involves this camera, any help would be appreciated.

### Are you modding a build from source or with Lua?

Source

### What is your build target?

Windows x64

### Did you edit anything in this build? If so, mention or summarize your changes.

No

**Comment by frantastic24** [comment]
Those are just warnings, they can be ignored

**Comment by gr1mslasher** [comment]
I have tried running/compiling the game with it, and can't. It cancels the compiling process: Sys_error("export/debug/windows/obj/Options.txt: Permission denied")
It's either that or I don't know how to run the compile command. I've been doing "lime test windows"

**Comment by frantastic24** [comment]
It could be your antivirus or bad files
Try disabling your antivirus and/or making a clean build

**Comment by gr1mslasher** [comment]
I did it with it off, still didn't work. For a clean build, you mean reinstall the github files for Psyche Engine and redo the installation process right?

**Comment by frantastic24** [comment]
Again the camera isn't a problem, it's that lime can't access your files for whatever reason
I'm assuming it works for html because it uses a different folder

**Comment by gr1mslasher** [comment]
So how do I get Lime to access my files? Should I redownload the ZIP folder for Psyche and redo the process of compiling it?
Another thing is that I want to make a mod with it's own executable like Mario Madness and Trollge Files, and in those mods you can't see the source code. If I compile Psyche would that make an executable with it's own icon and name in the files?

**Comment by gr1mslasher** [comment]
Update on the situation. Psyche works on html but not on windows, and you were right about not fixing the camera. It's this one line that prevents me from running it: 
Sys_error("export/release/windows/obj/Options.txt: Permission denied")
I went through the files from export to obj in the files and Options.txt is not there (the image link is a screenshot of what I see in obj). I tried adding a .txt file named Options to fix it but didn't work.
![image](https://user-images.githubusercontent.com/57470288/200135813-23cb2470-ad9c-45a4-984e-5f208be4d473.png)
